### ISSUE TITLE
using my 2d detector datasets, training precision is too low

### ISSUE THREAD

**Issue by yanqi1811** [paTRICK-swk/P-STMO#21]
hi, thank you for your work, this is nice!
I use my 2d detector which if fine-tuned in human3.6m, then I get new 2d points, such as cpn_ft_h36m_dbb.npz. But when I training the model using my 2d dataset, the  precision is too low. This is a log picture!

![20230112134419](https://user-images.githubusercontent.com/31723725/211988871-6e44f6e2-4210-4d16-ba2b-d7d62b09c8bf.png)

**Comment by paTRICK-swk** [comment]
1. Make sure you have run your 2d detector on both the training and test sets.
2. You can adjust the learning rate to see if it works.

**Comment by ovshake** [comment]
Hi @yanqi1811 , what format are you saving the 2D keypoints? Can you please help me here?
